commons-civil/src/main/java/uk/gov/hmcts/reform/civil/documentmanagement/SecuredDocumentManagementService.java

```diff
@@ -98,6 +99,57 @@ public CaseDocument uploadDocument(String authorisation, PDF pdf) {
         }
     }
 
+    @Retryable(value = {DocumentUploadException.class}, backoff = @Backoff(delay = 200))
+    @Override
+    public CaseDocument uploadDocument(String authorisation, UploadedDocument uploadedDocument) {
+
+        String originalFileName = uploadedDocument.getFileBaseName();
+        log.info("Uploading file {}", originalFileName);
+
+        try {
+            MultipartFile file
+                = new InMemoryMultipartFile(FILES_NAME, originalFileName, ALL_VALUE, uploadedDocument.getFile().getBytes());
+
+            DocumentUploadRequest documentUploadRequest = new DocumentUploadRequest(
+                Classification.RESTRICTED.toString(),
+                "CIVIL",
+                "CIVIL",
+                Collections.singletonList(file)
+            );
+
+            UploadResponse response = caseDocumentClientApi.uploadDocuments(
+                authorisation,
+                authTokenGenerator.generate(),
+                documentUploadRequest
+            );
+
+            Document document = response.getDocuments().stream()
+                .findFirst()
+                .orElseThrow(() -> new DocumentUploadException(originalFileName));
+
+            return CaseDocument.builder()
+                .documentLink(uk.gov.hmcts.reform.civil.documentmanagement.model.Document.builder()
+                                  .documentUrl(document.links.self.href)
+                                  .documentBinaryUrl(document.links.binary.href)
+                                  .documentFileName(originalFileName)
+                                  .documentHash(document.hashToken)
+                                  .build())
+                .documentName(originalFileName)
+                .createdDatetime(LocalDateTimeHelper.fromUTC(document.createdOn
+                                                                 .toInstant()
+                                                                 .atZone(ZoneId.systemDefault())
+                                                                 .toLocalDateTime()))
+                .documentSize(document.size)
+                .createdBy(CREATED_BY)
+                .build();
+
+        } catch (Exception ex) {
+            log.error("Failed uploading file {}", originalFileName, ex);
```

Comment:
now the controller advice has the error log we dont need here too, file name could be passed in to the DocumentUploadException and pulled when needed

Reply:
Removed the duplicate log